crates/uv/tests/lock.rs

```diff
@@ -173,8 +173,8 @@ fn lock_sdist_registry() -> Result<()> {
         [[distribution]]
         name = "project"
         version = "0.1.0"
-        source = "editable+file://[TEMP_DIR]/"
-        sdist = { url = "file://[TEMP_DIR]/" }
+        source = "editable+"
```

Comment:
This is ugly, do we know something better?

Reply:
Perhaps just `editable`? And adjust the parser to make a trailing `+` as optional (or even an error).

Reply:
I think my preference is not using `{}+{}` at all (either `editable = "."` or split this `source` into two fields), but for now i've added a `editable+.` as workaround.

Reply:
Is this tracked somewhere?

Reply:
Added an item to https://github.com/astral-sh/uv/issues/3611, i'm not yet deep enough here though to be clear on what the solution is.